### PR TITLE
More viro tweaks

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* tweak: Roundstart diseases are now flavor text only 80% of the time, with 10% chance of having a beneficial symptom, and 10% (down from 50%) chance of having a lesser annoying symptom (such as Loudness syndrome)
* tweak: Automated Sleeping Syndrome danger rating increased from annoying to hindrance, so it can no longer be a roundstart symptom.
* tweak: Catbeasts no longer suffer from their own disease's symptoms.
* tweak: Infected blood found in Urns or Bowls in Xenoarch now always contains beneficial symptoms.
* tweak: Pathogenic Clouds-emitting large artifacts now always contain deadly symptoms (accounting for the fact that those are pretty damn rare)
* tweak: Reduced the chance for infected maint mice to have the Contact spread vector from 60% to 40%. Mice without this vector can not spread their disease, but may still provide a virologist with some rare antigen and an extra sample.
* tweak: The isolation centrifuge now sends you a warning if you try to synthesize a vaccine with a duration longer than a minute or two.
* tweak: Glasgow syndrome can no longer load you with up to 325 units of alcohol, instead capping at the disease's strength times five, so up to 35.
* tweak: Mice infected with arachnogenesis now explode into 3 spiderlings, instead of vomitting too many of them over time.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Press F to pay respects to nu-viro cause it was fun while it lasted

**Comment by DeityLink** [comment]
Roundstart diseases were meant to be minor things you could easily ignore and get cured. but you can hardly ignore something like Automated Sleeping Syndrome, and MDs will have you rest in beds for hours because they're not sure how long it takes to cure.

With this PR they'll be more safe to ignore, without being gone altogether.

**Comment by tuypo1** [comment]
>Pathogenic Clouds-emitting large artifacts now always contain deadly symptoms

It doesn't seem as fun if they can't have good stuff.

**Comment by DeityLink** [comment]
@tuypo1 The Bowls and Urns filled with beneficial symptom infected blood are very easy to find, I get 2-3 per round on average when playing Xenoarch (from playing 3 rounds), while over the same rounds I only found 1 large artifact, which have so many possible effects that it's very unlikely to get one that spreads pathogen.

Deadly symptoms are much rarer, this way a traitor xenoarch is guarranted to get some use of a large artifact.

I hate how most xenoarch randomized large artifact most of the time don't do anything that can be remotely useful to either jobbies or antags due to their random nature or retarded activation requiring "precise atmospheric conditions" so at LEAST the symptoms should make up for the trouble of activating it.

**Comment by tuypo1** [comment]
I suppose so

**Comment by FudgePucker** [comment]
>precise atmospheric conditions

Just shout over coms "Boy I wish I could set up the specific atmospheric conditions for this artifact with benifital effects" and wait for a MoMMI to pop out of the vent

**Comment by DeityLink** [comment]
I took care of the secret repo conflict

**Comment by Kurfursten** [comment]
this PR needs to be aware of #23598 which will get merged first and has support

**Comment by DeityLink** [comment]
@Kurfursten I've removed the part that may cause conflicts.

File: code/modules/virus2/centrifuge.dm [hunk 1]
```diff
@@ -457,6 +457,9 @@
 
 			var/antibody = antibody_choices[choice]
 
+			if (antibodies[antibody] < 49)
```
Comment (Kurfursten) [inline code/modules/virus2/centrifuge.dm:460]: <=, or just <50

File: code/modules/virus2/disease2.dm [hunk 2]
```diff
@@ -444,6 +444,12 @@ var/global/list/disease2_list = list()
 			log += "<br />[timestamp()] destroyed enemy [enemy_pathogen.form] #[ID] ([strength] > [enemy_pathogen.strength])"
 			enemy_pathogen.cure(mob)
 
+	if (ishuman(mob))
+		var/mob/living/carbon/human/H = mob
+		if(H.species && H.species.name == "Tajaran")//Tajaran were born in the disease, molded by it.
```
Comment (Kurfursten) [inline code/modules/virus2/disease2.dm]: iscatbeast(mob), includes an ishuman check so you can remove that too and eschew the first if as well

**Comment by Kurfursten** [comment]
Besides those minor things, this looks good to go

File: code/modules/virus2/centrifuge.dm (hunk 1 above)
Comment (DeityLink) [reply code/modules/virus2/centrifuge.dm:460]: I put 49 since I considered a 2 minutes wait to still be acceptable.

Comment (Kurfursten) [reply code/modules/virus2/centrifuge.dm:460]: But your feedback text recommends 50

Comment (DeityLink) [reply code/modules/virus2/centrifuge.dm:460]: pulling the vial out to get another sample at concentration 50 may as well take an extra minute anyway. bother aren't exclusive.

**Comment by DeityLink** [comment]
@Kurfursten I added the iscatbeast() macro, is it good enough?

**Review by Kurfursten — APPROVED** [review]

**Pull request merged**